src/content/en/fundamentals/accessibility/00-intro/intro-four.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: shared/narrow
+title: "Screen Readers"
+description: "Introduction to screen reader software"
+published_on: 2016-03-01
+updated_on: 2016-03-01
+order: 4
+translation_priority: 0
+authors:
+  - megginkearney
+  - dgash
+key-takeaways:
+  tldr: 
+    - "Learn what accessibility means and how it applies to web development."
+    - "Learn how to make web sites accessible and usable for everyone."
+    - "Learn how to include basic accessibility with minimal development impace."
+    - "Learn what HTML features are available and how to use them to improve accessibility."
+    - "Learn about advanced accessibility techniques for creating polished accessibility experiences."
+notes:
+  efficiency:
+    - "Understanding the accessibility issue, its scope, and its impact can make you a better web developer."
+  problem-solving:
+    - "Catching, identifying, and removing potential accessibility roadblocks before they happen can improve your development process and reduce maintenance requirements."
+---
+
+Accessibility is about understanding the experience of a diverse range of users. One popular type of software is the *screen reader*, a program that enables visually impaired people to use computers by reading screen text aloud in a generated voice. The user can control what is read by moving the cursor to a relevant area with the mouse or keyboard.
+
+We asked Victor Tsaran to explain how, as a blind person, he accesses the web using a free screen reader for Windows called NonVisual Desktop Access, or NVDA.
+
+>I access the web using a screen reader that converts text and UI elements to descriptive text in either speech or braille form. For example, I would use NVDA to access the BBC.com website.
```

Comment:
@dgash @alice

I feel like in this case we really need to include the video (https://www.youtube.com/watch?v=QW_dUs9D1oQ) showing Victor using his screen reader, because it's so hard to convey the experience just in text alone.

Unfortunately we went off script here and Victor used VoiceOver on Mac with Google Groups instead of reading through BBC :\